### PR TITLE
[bddisasm] Update to 1.34.7

### PR DESCRIPTION
**Pull request by ianichitei**
This updates the port of bddisasm to version 1.34.7.

I used 0 as the `port-version` for 1.34.7, but I'm not entirely sure if this is correct.

### CODE REVIEW

File: ports/bddisasm/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bddisasm",
-  "version": "1.34.2",
+  "version": "1.34.7",
   "port-version": 1,
```
Comment (JonLiu1993) [inline ports/bddisasm/vcpkg.json:4]:
```suggestion
```

Comment (JonLiu1993) [reply ports/bddisasm/vcpkg.json:4]: @ianichitei ,Thanks for your pr, when we update port, we should set "port-version" to 0 or remove "port-version"

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by ianichitei** [comment]
So the bot makes things a bit confusing. I first had `port-version` set to 0, but it suggested a change to `1`. Was that something like a false positive from the bot?

**Comment by JonLiu1993** [comment]
> So the bot makes things a bit confusing. I first had `port-version` set to 0, but it suggested a change to `1`. Was that something like a false positive from the bot?

Maybe it misled you. when we update the port, we need to reset the port-version to 0 or delete it. this is correct.

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**